No longer install the Jupyter extension by default (#20218)
microsoft/vscode-python: README.md

## Installed extensions

> The Python extension will automatically install the [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) and [isort](https://marketplace.visualstudio.com/items?itemName=ms-python.isort) extensions to give you the best experience when working with Python files. However, they are both optional dependencies, meaning the Python extension will remain fully functional if either one fails to be installed. You can [uninstall](https://code.visualstudio.com/docs/editor/extension-marketplace#_uninstall-an-extension) Pylance at the expense of some features if you’re using a different language server.

Reviewer: I'm curious why you call out "if either one fails to be installed" here, is that just about a case where the marketplace installation fails? Does that ever happen?

Reviewer 2: We have had issues where installing Python itself has failed (typically corrupted download or unzip somehow left files out). We could reword this to:

```suggestion
The Python extension will automatically install the [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) and [isort](https://marketplace.visualstudio.com/items?itemName=ms-python.isort) extensions to give you the best experience when working with Python files. However, they are both optional dependencies, meaning the Python extension will remain functional if either one is not installed. You can [uninstall](https://code.visualstudio.com/docs/editor/extension-marketplace#_uninstall-an-extension) Pylance at the expense of some features if you’re using a different language server.
```

> The Python extension will automatically install the [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) and [isort](https://marketplace.visualstudio.com/items?itemName=ms-python.isort) extensions to give you the best experience when working with Python files. However, they are both  optional dependencies, meaning the Python extension will remain fully functional if either one fails to be installed. You can [uninstall](https://code.visualstudio.com/docs/editor/extension-marketplace#_uninstall-an-extension) Pylance at the expense of some features if you’re using a different language server.

Reviewer 2: ```suggestion
The Python extension will automatically install the [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) and [isort](https://marketplace.visualstudio.com/items?itemName=ms-python.isort) extensions to give you the best experience when working with Python files. However, they are both optional dependencies, meaning the Python extension will remain fully functional if either one fails to be installed. You can [uninstall](https://code.visualstudio.com/docs/editor/extension-marketplace#_uninstall-an-extension) Pylance at the expense of some features if you’re using a different language server.
```
